README.md

```diff
@@ -30,236 +33,171 @@
 [ ![troubleshooting-shield] ][troubleshooting]
 [ ![migration-shield][] ][migration]
 
-
 # JDA (Java Discord API)
 
-JDA strives to provide a clean and full wrapping of the Discord REST API and its WebSocket-Events for Java.
-This library is a helpful tool that provides the functionality to create a Discord bot in Java.
+This open source library is intended for implementing bots on Discord using the real-time gateway and REST API. It provides event based functionality to implement bots of any kind, allowing for effective and scalable applications.
 
-## Summary
+## Overview
 
-1. [Introduction](#creating-the-jda-object)
-2. [Sharding](#sharding-a-bot)
-3. [Entity Lifetimes](#entity-lifetimes)
-4. [Download](#download)
-5. [Documentation](#documentation)
-6. [Support](#getting-help)
-7. [Extensions And Plugins](#third-party-recommendations)
-8. [Contributing](#contributing-to-jda)
-9. [Dependencies](#dependencies)
-10. [Other Libraries](#related-projects)
+The core concepts of JDA have been developed to make building scalable apps easy:
 
-## UserBots and SelfBots
+1. Event System  
+    Providing simplified events from the gateway API, to respond to any platform events in real-time without much hassle.
+2. Rest Actions  
+    Easy to use and scalable implementation of REST API functionality, letting you choose between callbacks with combinators, futures, and blocking.
```

Comment:
Wouldn't it be worth mentioning, that the Library also handles and respects discord rate limits, so that you don't have to yourself?
Just an idea.